jetty-websocket/javax-websocket-server/src/main/java/org/eclipse/jetty/websocket/javax/server/JavaxWebSocketServerContainer.java

```diff
@@ -283,20 +311,21 @@ public long getDefaultAsyncSendTimeout()
     public int getDefaultMaxBinaryMessageBufferSize()
     {
         // TODO: warn on long -> int conversion issue
-        return (int)this._webSocketCreatorMapping.getDefaultMaxBinaryMessageSize();
+        // TODO: Should this be Filter?
+        return (int)customizer.getMaxBinaryMessageSize();
     }
 
     @Override
     public long getDefaultMaxSessionIdleTimeout()
     {
-        return this._webSocketCreatorMapping.getDefaultIdleTimeout().toMillis();
+        return customizer.getIdleTimeout().toMillis();
```

Comment:
Whut?

Reply:
The configuration is now applied by one or more customizers - that know about global defaults and tri-state any set configuration.
In this case rather than applying defaults from the mapping, we have a set a customizer for the container (the base defaults) and then another for each mapping (rather than one for just the mapping itself)